hoodie-common/src/main/java/com/uber/hoodie/common/util/FSUtils.java

```diff
@@ -109,7 +110,12 @@ public static String getCommitFromCommitFile(String commitFileName) {
     }
 
     public static String getCommitTime(String fullFileName) {
-        return fullFileName.split("_")[2].split("\\.")[0];
+        Matcher matcher = LOG_FILE_PATTERN.matcher(fullFileName);
+        if (!matcher.find()) {
+            return fullFileName.split("_")[2].split("\\.")[0];
+        } else {
+            return fullFileName.split("_")[1].split("\\.")[0];
```

Comment:
Can you try using the FileGroup & FileSlice abstracts to implement this more cleanly..